assets/js/googlesitekit/constants.js

```diff
@@ -52,3 +52,11 @@ export const SITE_KIT_VIEW_CONTEXTS = [
 	VIEW_CONTEXT_SETTINGS,
 	VIEW_CONTEXT_MODULE_SETUP,
 ];
+
+// Site Kit view-only contexts
+export const SITE_KIT_VIEW_ONLY_CONTEXTS = [
+	VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
+	VIEW_CONTEXT_ENTITY_DASHBOARD_VIEW_ONLY,
+	VIEW_CONTEXT_ADMIN_BAR_VIEW_ONLY,
+	VIEW_CONTEXT_WP_DASHBOARD_VIEW_ONLY,
+];
```

Comment:
Since we're defining this here, let's update https://github.com/google/site-kit-wp/blob/e572e5972389524b1a28c635bf8feec261d6b68d/assets/js/hooks/useViewOnly.js#L40-L45 to use this same array, so the logic is at least a bit less duplicated and things don't go out-of-sync if/when we update this array 😄

Reply:
@jimmymadon and I chatted on Slack and he'll fix this in the PR for #9071 👍🏻